src/api/todos.ts

```diff
@@ -0,0 +1,8 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+// Add more methods here
```

Comment:
it was hint that api functions should be here

Reply:
I forgot to look at this file at all.